brute_force_detector/brute_force_detector.cpp

```diff
@@ -205,59 +198,53 @@ int main(int argc, char **argv)
     char *whitelistFilePath = NULL;
     bool whitelistParserVerbose = false;
     bool RDP = false, SSH = false, TELNET = false;
-    while((opt = TRAP_GETOPT(argc, argv, module_getopt_string, long_options)) != -1)
-    {
-        switch (opt)
-        {
-        case 'c': //config
-            configFilePath = optarg;
-            break;
-        case 'w':
-            whitelistFilePath = optarg;
-            break;
-        case 'W':
-            whitelistParserVerbose = true;
-            break;
-        case 'R':
-            RDP = true;
-            break;
-        case 'S':
-            SSH = true;
-            break;
-        case 'T':
-            TELNET = true;
-            break;
-        default:
-            cerr << "Error: Invalid arguments.\n";
-            FREE_MODULE_INFO_STRUCT(MODULE_BASIC_INFO, MODULE_PARAMS)
-            return 3;
+    while ((opt = TRAP_GETOPT(argc, argv, module_getopt_string, long_options)) != -1) {
+        switch (opt) {
+            case 'c': // config
```

Comment:
This change is not compliant with coding style: https://nemea.liberouter.org/developer/coding-style-c#statements